src/components/ChatList/ChannelListHeaderItem.tsx

```diff
@@ -4,11 +4,12 @@ import {StyleSheet, Text, View} from 'react-native';
 
 import dimen from '../../utils/dimen';
 import {SIGNED} from '../../hooks/core/constant';
-import {colors} from '../../utils/colors';
 import {fonts, normalizeFontSize} from '../../utils/fonts';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to import `COLORS` from the `utils/theme` module instead of importing `colors` from the `utils/colors` module. This change is consistent with the PR summary and seems to be part of a larger refactoring effort to standardize color usage across the codebase.
```diff
- import {colors} from '../../utils/colors';
+ import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->